modules/timepoint_list/php/timepoint_list.class.inc

```diff
@@ -300,5 +318,85 @@ class Timepoint_List extends \NDB_Menu
             )
         );
     }
-}
+    /**
+     * Handle archive a visitlabel
+     *
+     * @param ServerRequestInterface $request The PSR15 ServerRequestInterface.
+     *
+     * @return \LORIS\Http\Response\JSON\OK
+     */
+    protected function handleDelete(
+        ServerRequestInterface $request
+    ): ResponseInterface {
+        $body     = $request->getBody()->getContents();
+        $jsonBody = json_decode($body, true);
+        $id       = null;
+        if (is_array($jsonBody) && array_key_exists('sessionID', $jsonBody)) {
```

Comment:
I think you're better off checking if it's not set and returning an error, then retrieving the value and getting it.

Otherwise you're still running the update session with a value of `ID=null` in the where clause. I don't think it will ever affect any rows since the ID is a primary key, but it still seems like a dangerous assumption to make if the code changes in a way that something else happens here.